### ISSUE TITLE
Multiple `it`s within `context` are not evaluated

### ISSUE THREAD

**Issue by steakknife** [Quick/Quick#324]
Running this will not encounter and of the assertions.

```swift
// Swift 1.2
import Quick // pod 0.3.1
import Nimble // pod 0.4.2

class SomeSpec: QuickSpec {
	override func spec() {
		describe("frobulator") {
			context("within tolerance") {
				it("something + whatever") {
					expect(code).to(equal(something)) // works great
				}
				it("something - whatever") {
					assertionFailure("1") // never happens
					expect(other code).to(equal(whatever))
				}
			}
			context("outside tolerance") {
				it("blah + foo") {
					expect(code).to(equal(value)) // works great
				}
				it("blah - foo") {
					assertionFailure("2") // never happens
					expect(code).to(equal(value))
				}
			}
		}
	}
}
```

**Comment by steakknife** [comment]
It looks like the example group name deciding code isn't comparing `String`s precisely and decides to throw away the second one silently.

**Comment by jeffh** [comment]
Hey @steakknife, this looks like a similar issue to #299. An implementation detail of Quick is that it converts examples to XCTest methods. Since test methods don't support arbitrary characters, Quick does some character stripping, which makes both examples map to the same test method.

It's definitely an issue that needs to get address though. Thanks for filing it :+1:. For now, a work around is to have ascii characters that differentiate the examples.

**Comment by romankl** [comment]
Looks like 41ce3fe (available with Swift 1.2/ Swift 2.0) solved the problem.

**Comment by jeffh** [comment]
Thanks for pinging @romankl :+1:. Closing.
